components/helper/ShareButtons.tsx

```diff
@@ -9,9 +11,133 @@ import {
   TwitterShareButton,
 } from 'react-share';
 
-export default function ShareButtons() {
+type ShareButtonsProps = {
+  popup: boolean;
+  setCopied: React.Dispatch<React.SetStateAction<boolean>>;
+  copied: boolean;
+};
+
+export default function ShareButtons({
+  popup, setCopied, copied,
+}: ShareButtonsProps) {
   const url = 'https://clearviction.org/calculator/head-initial-1-cont';
 
+  const style = {
+    fontSize: '1.25rem',
+    fontWeight: 400,
+    lineHeight: '1.375rem',
+    letterSpacing: '.0156rem',
+    textAlign: 'left',
+    color: '#2F3554',
+    textDecoration: 'underline',
+    marginLeft: '.4375rem',
+  };
+
+  const handleCopyClick = async () => {
+    await navigator.clipboard.writeText(url);
+    setCopied(true);
+  };
+
+  function handleEmailClick() {
+    const body = encodeURIComponent(url);
+    const mailtoUrl = `mailto:?body=${body}`;
+    window.open(mailtoUrl);
+  }
+
+  if (copied) {
+    return (
+      <Box
+        className="copied-message-box"
+        sx={{
+          width: '20.3125rem',
+          height: '5.9375rem',
+          padding: '1rem',
+          borderRadius: '.375rem',
```

Comment:
For things like border radius, or borders, we can use px, because we probably want to keep that styling from growing when the component grows. There are a few cases of that, can you please adjust them.

Reply:
Makes sense